### PR TITLE
Adding test for the logout link

### LINKED ISSUE #108: Write a test that confirms functionality of the logout link
At demo3, a mentor suggested we add a test that confirms the functionality of the logout link (and not just the cookie).

### PR DESCRIPTION
**Pull request by DanielMy777**
Adding a single test that confirms the user has been logged out after he clicked 'logout'.
fix #108

### CODE REVIEW

**Review by Micha546 — COMMENTED** [review]
💯

File: main/tests.py [hunk 1]
```diff
@@ -59,3 +59,10 @@ def test_invalid_login_info(self, client, make_user):
         response = client.post('/login/', credentials, follow=True)
         assert response.status_code == 200
         assert b"Please enter a correct email" in response.content
+
+    def test_logout_link_functionality(self, client, make_user):
+        sample_user = make_user("test1@gmail.com", "testy", "test", "1995-05-05", "test123")
+        client.login(email="test1@gmail.com", password="test123")
+        response = client.get('/logout/')
+        logged_user = response.wsgi_request.user
```
Comment (tamirho) [inline main/tests.py:67]: Just for my knowledge...
`response.wsgi_request.user` spouse to be `None`?

**Comment by tamirho** [comment]
LGTM

File: main/tests.py (hunk 1 above)
Comment (DanielMy777) [reply main/tests.py:67]: Hey @tamirho 
Not `None`, But a special 'Anonnymous user' object when no user is logged in.
You can read more about it here:
https://docs.djangoproject.com/en/3.2/ref/contrib/auth/#anonymoususer-object

Comment (tamirho) [reply main/tests.py:67]: Nice! Thanks

**Review by simzacks — APPROVED** [review]

**Review by OrAsaf — APPROVED** [review]

**Pull request merged**